modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-disable.adoc

```diff
@@ -0,0 +1,68 @@
+= rpk cluster partitions disable
+:description: rpk cluster partitions disable
```

Comment:
These descriptions should describe the command (essentially include the content of the first paragraph on the page.

They are used in search engines as well as our index pages to tell users more about the page. Duplicating the command is not useful.